app/models/arbitrary_assignment_creation.rb

```diff
@@ -1,11 +1,14 @@
 class ArbitraryAssignmentCreation
+  attr_reader :visitor_id, :split_name, :variant, :bulk_assignment_id, :context
+
   def initialize(params = {})
     @visitor_id = params[:visitor_id]
     @split_name = params[:split_name]
     @variant = params[:variant]
     @mixpanel_result = params[:mixpanel_result]
     @bulk_assignment_id = params[:bulk_assignment_id]
     @context = params[:context]
+    @force = params[:force] if params.key?(:force)
```

Comment:
why the condition here?

Reply:
Because I'm distinguishing between not-passed and expressly-set-to-null so I can default to false.